### PR TITLE
Add new static inline functions

### PR DESCRIPTION
**Pull request by dralley**

### CODE REVIEW

File: libsolv-sys/src/export/pool.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@ use libc::{c_int, c_char};
 
 extern "C" {
     pub fn e_pool_id2solvable(pool: *const Pool, p: Id) -> *mut Solvable;
+    pub fn e_pool_solvable2id(pool: *const Pool, p: *const Solvable) -> Id;
     pub fn e_pool_solvid2str(pool: *mut Pool, p: Id) -> *const c_char;
     pub fn e_pool_match_nevr(pool: *mut Pool, s: *mut Solvable, d: Id) -> c_int;
     pub fn pool_whatprovides(pool: *mut Pool, d: Id) -> Id;
```
Comment (dralley) [inline libsolv-sys/src/export/pool.rs:9]: I'm a little confused about why this one and the one below it doesn't have a prefix

**Comment by dralley** [comment]
Should all the static inline functions be exposed or just a subset?  Ones in other files not added here thus far?

**Comment by ambaxter** [comment]
In theory all of the publicly accessible ones should be exposed, but IIRC I was limiting myself to the ones that were immediately necessary for the work I was doing.

**Comment by dralley** [comment]
Gotcha.  These were all added in the past 18 months so they wouldn't have existed at the time.  That probably means they're not particularly core to anything, though.

**Pull request merged**